src/jvm/test/org/jetbrains/kotlinx/lincheck/test/representation/StateRepresentationTest.kt

```diff
@@ -65,7 +65,7 @@ open class ModelCheckingStateReportingTest {
         check("STATE: 0" in log)
         check("STATE: 1" in log)
         check("STATE: 4" in log)
-        check(log.split("incrementAndGet(): 1").size - 1 == 1) { "A method call is logged twice in the trace" }
+        check(logWithoutVerbosePart(log).split("incrementAndGet(): 1").size - 1 == 1) { "A method call is logged twice in the trace" }
```

Comment:
Can we replace all these `check`s with a single check with the expected output?

Reply:
Done